templates/sms/invoice_notice_fi.txt

```diff
@@ -1,2 +1,8 @@
-Hyvä asiakas. Tässä linkki tuotteen {{ product_name }} maksamiseen tai sen irtisanomiseen seuraavalle kaudelle. Maksathan laskun {{ due_date }} mennessä.
+Hyvä asiakas. Tässä linkki tuotteen {{ product_name }} maksamiseen tai sen irtisanomiseen seuraavalle kaudelle. Maksathan laskun {{ order.due_date.strftime("%d.%m.%Y") }} mennessä.
 {{ payment_url }}
+{% if new_berth_order %}
```

Comment:
I would show the information in both versions, new or not, because people can have multiple boats and berths. It also gives more context to the message.

Reply:
Afaik, this message also gets sent e.g. when there's a winter storage related message which has `WinterStorageLease` and that doesn't contain berth information.